File: Vinyl/Vinyl/Models/Song.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Vinyl.Models
{
    public class Song
    {
        [Key]
        public int vId { get; set; }
        public string vArtist { get; set; }
        public string vAlbum { get; set; }
        public string vBarcode { get; set; }
        public string vCatalogNo { get; set; }
        public string vLastReleaseYear { get; set; }
        public string vReleaseYear { get; set; }
        public string vFormat { get; set; }
        public string vLabel { get; set; }
        public string vCountry { get; set; }
        public string vTop { get; set; }
        public string vState { get; set; }
        public string vMusicalStyle { get; set; }
        public string vMarks { get; set; }
        public int vPrice { get; set; }
    }
}

File: vinyl-store/src/app/shared/song.ts
export class Song {
    vId: number;
    vArtist: string;
    vAlbum: string;
    vBarcode: string;
    vCatalogNo: string;
    vLastReleaseYear: string;
    vReleaseYear: string;
    vFormat: string;
    vLabel: string;
    vCountry: string;
    vTop: string;
    vState: string;
    vMusicalStyle: string;
    vMarks: string;
    vPrice: number;

    constructor(vId: number, vArtist: string, vAlbum: string, vBarcode: string, vCatalogNo: string,
        vLastReleaseYear: string, vReleaseYear: string, vFormat: string, vLabel:string, vCountry: string,
        vTop:string, vState: string, vMusicalStyle: string, vMarks: string, vPrice: number) {
        this.vId = vId;
        this.vArtist = vArtist;
        this.vAlbum = vAlbum;
        this.vBarcode = vBarcode;
        this.vCatalogNo = vCatalogNo;
        this.vLastReleaseYear = vLastReleaseYear;
        this.vReleaseYear = vReleaseYear;
        this.vFormat = vFormat;
        this.vLabel = vLabel;
        this.vCountry = vCountry;
        this.vTop = vTop;
        this.vState = vState;
        this.vMusicalStyle = vMusicalStyle;
        this.vMarks = vMarks;
        this.vPrice = vPrice;
    }
}